internal/pack.h

```diff
@@ -237,6 +237,32 @@ class ScalarRoundingOffsetGenerator<RoundingMode::ProbabilisticXorshift> {
   std::uint8_t x_;
 };
 
+// A RoundingOffsetGenerator based on an 8-bit add/mod
+// low-discrepancy sequence.  See less-than-8-bit.txt for
+// an explanation (the constant 97 is important - it must
+// be both relatively prime to 255, in order for the sequence
+// to be full-period, and c/255 should be close to 0.38 to
+// obtain low discrepancy).  Uses a small bit hack to avoid
+// expensive % operations.
+template <>
+class ScalarRoundingOffsetGenerator<RoundingMode::ProbabilisticAddmod> {
+  const uint8_t AddConst = 97;
```

Comment:
"static const" please - so we get a plain old compile-time constant and give the compiler a green light to optimize it away and use the literal initializer value instead --- since it would be UB to ever modify it.